charts/internal/events/values.yaml

```diff
@@ -1,13 +1,20 @@
+global:
+  observe:
+    customer: "123456789012"
```

Comment:
Can you please remove this global block -- the customer ID is used to generate URLs, which is now done with a template include. global.observe.customer is still a supported way to configure the chart, for backwards-compatibility, but it's now deprecated in favor of `collectionEndpoint`.

Reply:
Hey @obs-gh-jeremyjackins The only reason I set it is that it is "required" in order to run a `helm template` or `helm lint`. If I don't have a default value, even a placeholder like I did, these commands can't be executed properly. If you prefer I can set a default placeholder for the `collectionEndpoint` instead of this.